packages/network/src/workers/constants.ts

```diff
@@ -6,4 +6,4 @@ export enum SyncState {
   LIVE,
 }
 
-export const GodID = "0x060D" as EntityID;
+export const GodID = "0x060d" as EntityID;
```

Comment:
why this change?

Reply:
the ID comes back from on-chain lowercase